README.md

```diff
@@ -1,44 +1,49 @@
-# Week 10
+#Week 10
 
-## Mongoose
+##Mongoose
 
-### Question #1
+###Question #1
 
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
-```text
-Your answer...
+*SQL: Relational, structured, (tables, columns, rows, records), structure predefined and inflexible.
+NoSQL: Non relational, non-structured (BSON key-value pairs, documents ), structure flexible and easy to change.*
+
+*SQL is better for structured, defined, discrete items with exact specifications.   NoSQL is better for less structured environments that might need flexibility to change.*
 
-```
 
-### Question #2
+###Question #2
 
-What's wrong with this mongoose code and how might we fix it?
-(Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
+What's wrong with this mongoose code and how might we fix it? (Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
 
 ```js
 var results = AuthorModel.find({name: "Bob"});
 console.log(results);
 ```
 
+*Mongoose has no "find" method; mongoose uses findOne, and the syntax is a little different.  The following should work:*
+
 ```js
-// Your answer...
+var results = AuthorModel.findOne({'name': "Bob"});
+console.log(results);
```

Comment:
The issue is that `.find` is asynchronous, meaning that `console.log(results)` must be done inside of a callback or promise.